index.html

```diff
@@ -241,7 +143,8 @@ <h3 class="nav-sub-sub-heading weight--500 text--info mt-12">
                     </aside>
 
                     <aside>
-                        <form action="https://formspree.io/f/xknedbwy" method="post" class="contact-form mt-24">
+                        <form id="validateForm" action="https://formspree.io/f/xknedbwy" method="post"
+                            class="contact-form mt-24">
                             <label for="" class="d-none">Full name</label>
                             <input type="text" maxlength="30" required name="fullName" placeholder="your full name"
                                 class="weight--400">
```

Comment:
- [ ] [**OPTIONAL**] Kindly make sure you correct this title, because your portfolio will probably be viewed by many people :+1: 

![Screenshot from 2022-12-23 15-25-53 (1)](https://user-images.githubusercontent.com/80458199/209348440-00cb4838-8256-4cd1-a0f1-8ebfcc5a28db.png)